### ISSUE TITLE
"Landing Gear" Hotkey not working

### ISSUE THREAD

**Issue by axel37** [tsunamayo/Skywanderers#613]
Hello !
I added brand new landing gear to my ship !

When connected to a switch, it works as expected. So,, I placed a hotkey brick, set it to "landing gear" then linked it to my ship's control !
"L"... nothing happens !
So I changed the hotkey to "Secondary Fire" in case something was wrong with my build...
"Mouse 2"... the landing gear deploys !

I did checked my keybinds, so it's not the issue.

**Comment by tsunamayo** [comment]
It works for me. Are you entering your spaceship core? Could you send me your blueprint so I check it out.
Cheers

**Comment by axel37** [comment]
I tried the same thing today and... now, I can't deploy the landing gear at all. I didn't touch anything since I created the issue... but here's the blueprint.

[Axools Junkship.zip](https://github.com/tsunamayo/Skywanderers/files/1847150/Axools.Junkship.zip)

_It's not the first time something like this happens to me in this game. One day it works (or doesn't), and the day after, it just stops working, without me touching anything..._
